packages/tokens/README.md

```diff
@@ -13,7 +13,7 @@ Distribute design tokens to all Coveo products that rely on the Plasma Design Sy
 ## Installation
 
 ```bash
-npm install @vapor/tokens
+npm install @coveord/plasma-tokens
```

Comment:
pnpm?

Reply:
not all projects use pnpm, npm is the base standard that everyone knows and understand